source/mainnet/net/resources/glossary.rst

```diff
@@ -284,7 +291,7 @@ Finalization
 ============
 
 The process by which a block is marked to be "finalized", i.e. part of the
-authoritative :ref:`chain<glossary-chain>`. Transactions that are part of finalized blocks are considered authoritative. New blocks can be only added following the last finalized block. The finalization process is conducted periodically by the bakers with :ref:`staked amount<glossary-staked-amount>` at least 0.1% of the total amount of existing CCD.
+authoritative :ref:`chain<glossary-chain>`. Transactions that are part of finalized blocks are considered authoritative. New blocks can be only added following the last finalized block. The finalization process is conducted periodically by the bakers with a staked amount of at least 0.1% of the total amount (stake) of CCD in pools, known as the Finalization committee. Total stake in pools is referred to as total stake in pools without Passive Delegation.
```

Comment:
The finalization is no longer happening "periodically", but instead it's a "built-in" feature of the protocol. Without finalization then there would be no progress on the chain, i.e. an finalization has to happen for each round otherwise we do not proceed to the next round.

Before it was a protocol running on top of the normal consensus protocol.